docs/ConfigFile.md

```diff
@@ -26,7 +26,8 @@ Below is an example of the NNCF configuration file
                 },
             ],
         },
-        "hw_config_type": "cpu", // If specified, the compression algorithms will use parameter presets that are more likely to result in best performance on a given HW type. Optional.
+        "target_device": "VPU", // The target device, the specificity of which will be taken into account when compression in order to obtain the best performance for this type of device. The default "ANY" means compatible quantization supported by any HW. The parameter can takes values from the set ('CPU', 'GPU', 'VPU', 'ANY', 'NONE'). If you don't want to have any constraints on quantizer parameters you should set 'NONE' value.Optional.
```

Comment:
`when compression` -> `while compressing`, `can takes` -> `takes`, `set 'NONE' value` -> `set this value to 'NONE'`

Reply:
Done.